src/Sarif.Viewer.VisualStudio/FixSuggestedActionsSourceProvider.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft. All rights reserved. 
+// Licensed under the MIT license. See LICENSE file in the project root for full license information. 
+
+using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.Language.Intellisense;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.Text.Operations;
+using Microsoft.VisualStudio.Utilities;
+
+namespace Microsoft.Sarif.Viewer
+{
+    [Export(typeof(ISuggestedActionsSourceProvider))]
+    [Name(FixActionCategoryName)]
+    [ContentType("text")]
+    internal class FixSuggestedActionsSourceProvider : ISuggestedActionsSourceProvider
+    {
+        internal const string FixActionCategoryName = "SARIF fix suggestion";
```

Comment:
> "SARIF fix suggestion" [](start = 53, length = 23)

Put in RESX please

Reply:
Is there a way to use string resources in attribute arguments? Thats's where this is needed, in Line 14 above.

---
In reply to: [509476895](https://github.com/microsoft/sarif-visualstudio-extension/pull/228#discussion_r509476895) [](ancestors = 509476895)